examples/steel-puppeteer-starter/README.md

```diff
@@ -4,6 +4,7 @@ This template shows you how to use Steel with Puppeteer to run browser automatio
 
 [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz.svg)](https://stackblitz.com/edit/steel-puppeteer-starter?file=README.md)
 [![Open Replit Template](https://replit.com/badge/github/@steel-dev/steel-puppeteer-starter)](https://replit.com/@steel-dev/steel-puppeteer-starter?v=1)
+[![Open Val Town Template](https://stevekrouse-badge.web.val.run/?3)](https://www.val.town/v/stevekrouse/steel_puppeteer_starter)
```

Comment:
The `?3` is simply to get the cached refreshed as I edited this badge generator a number of times. It can be any path or query param there